src/util_libc.rs

```diff
@@ -5,9 +5,30 @@
 // <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+extern crate std;
 
 use crate::util::LazyUsize;
+use crate::Error;
 use core::ptr::NonNull;
+use std::io;
+
+pub fn fill_exact(mut buf: &mut [u8], f: impl Fn(&mut [u8]) -> libc::ssize_t) -> Result<(), Error> {
```

Comment:
Why `libc::ssize_t` instead of `isize`?

Reply:
This is mainly because all the functions I'm using this with `getrandom(2)`, `read(2)`, etc.. All return a `libc::ssize_t`, so using that type seemed less confusing. As you correctly point out, it doesn't really matter except for code clarity.